### PR TITLE
feat: full width for stick at bottom containers

### PR DESCRIPTION
**Pull request by AntoineAugusti**
Make sure containers with the class `js-stick-at-bottom-when-scrolling` are full width. Previously they were only the size of the container.

Made sure that the container is resized when the window width changes

Trello: https://trello.com/c/ooQJsbYS/531-increase-division-line-for-persistent-page-buttons

### CODE REVIEW

File: app/assets/javascripts/stick-to-window-when-scrolling.js [hunk 1]
```diff
@@ -789,6 +789,8 @@
 
       el.removeStickyClasses(this);
       $el.css('width', '');
+      $el.css('left', '');
+      $el.css('padding-left', '');
```
Comment (AntoineAugusti) [inline app/assets/javascripts/stick-to-window-when-scrolling.js:793]: Remove new attributes when releasing the sticky container

File: app/assets/javascripts/stick-to-window-when-scrolling.js [hunk 2]
```diff
@@ -929,6 +931,14 @@
   stickAtBottom.windowNotPastScrollingTo = function (windowPositions, scrollingTo) {
     return windowPositions.bottom > scrollingTo;
   };
+  stickAtBottom.setElWidth = function (el) {
+    var $el = el.$fixedEl;
+    $el.css({
+        // element will be absolutely positioned so cannot rely on parent element for width
+      'width': window.innerWidth + 'px',
+      'padding-left': (window.innerWidth - document.getElementsByTagName('main')[0].offsetWidth) / 2 + 'px',
```
Comment (AntoineAugusti) [inline app/assets/javascripts/stick-to-window-when-scrolling.js:939]: The width is the size of the window, but make sure the content stays in the middle by adding a left padding

File: app/assets/javascripts/stick-to-window-when-scrolling.js [hunk 3]
```diff
@@ -929,6 +931,14 @@
   stickAtBottom.windowNotPastScrollingTo = function (windowPositions, scrollingTo) {
     return windowPositions.bottom > scrollingTo;
   };
+  stickAtBottom.setElWidth = function (el) {
```
Comment (AntoineAugusti) [inline app/assets/javascripts/stick-to-window-when-scrolling.js:934]: Called automatically when the window is resized

File: tests/javascripts/collapsibleCheckboxes.test.js [hunk 4]
```diff
@@ -40,19 +40,21 @@ describe('Collapsible fieldset', () => {
...
-            Folders this team member can see
-          </legend>
-          <div class="checkboxes-nested">
-            <ul>
-              ${_checkboxes(1, 10)}
-            </ul>
-          </div>
-        </fieldset>
-      </div>`;
+      `<main>
```
Comment (AntoineAugusti) [inline tests/javascripts/collapsibleCheckboxes.test.js:43]: Needs to be wrapped in a `<main>` like other JS tests

**Review by jimleroyer — APPROVED** [review]
Good stuff, not always easy to pick on 3rd party libraries and fix something in there.

**Review by e-pang — COMMENTED** [review]
This is looking much better already, thanks! Can we make a couple of refinements? 
- Remove the fade (or speed it up) so that the overlapping UI is seen.
- Left align the sticky button with the non-sticky button (this will help with the jumpiness).
Let me know if you have any questions.

**Pull request merged**